test/setup/main.tf

```diff
@@ -26,9 +26,36 @@ module "project" {
   skip_gcloud_download = true
 
   activate_apis = [
+    "cloudkms.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "bigquery.googleapis.com",
     "bigquerystorage.googleapis.com",
     "serviceusage.googleapis.com"
   ]
 }
+
+#Can't delete this with Terraform.
+module "kms_keyring" {
+  source     = "terraform-google-modules/kms/google"
+  version    = "~> 1.2"
+  depends_on = [module.project] #waits for API config
+
+  project_id      = module.project.project_id
+  location        = "us"
+  keyring         = "ci-bigquery-keyring"
+  keys            = ["foo"]
+  prevent_destroy = "false"
+}
+
+module "initialize_encryption_account" {
+  #pinned to prevent bug with random cache count evaluation, see #82
```

Comment:
Thanks for flagging https://github.com/terraform-google-modules/terraform-google-gcloud/issues/82?
I see that we are using the latest CI image with TF 0.13 and probably why this surfaced here.
https://github.com/terraform-google-modules/terraform-google-bigquery/blob/862b8b6c1e5dfbbce57d4d585fb42eb82a68b980/build/int.cloudbuild.yaml#L41

Reply:
Is there are action to take here?